x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/validation_step/validation_step.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { EuiLoadingSpinner, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ml } from '../../../../../services/ml_api_service';
```

Comment:
for new code we should be using the context which provides this.
```
import { useMlApiContext } from '../../../../../contexts/kibana';
```

Reply:
Updated in a7e5fc69c41f0b104667516fc53457eebf0a649d